packages/backend/src/models/flow.ts

```diff
@@ -11,6 +11,7 @@ import ExtendedQueryBuilder from './query-builder'
 import Step from './step'
 import User from './user'
 
+export type NotificationFrequency = 'once' | 'always'
```

Comment:
Can we remove this? Duplicating the same thing from `IFlow` is a bad idea (2 sources of truth).

Code which needs this can obtain it directly from `IFlowErrorConfig` - e.g. `IFlowErrorConfig['notificationFrequency']`